faq.md

```diff
@@ -287,16 +287,8 @@ possible to create your own set of widgets using our base classes.
 
 ## Does Flutter work with any editors or IDEs?
```

Comment:
I don't think that makes sense on the medium to long term horizon (hopefully very fast the amount of users who even knew of that existing will be tiny fraction).

I suggest a email announcement to the dev group is a better way of handling it.